docs/production/deploying-to-cloud/gcp/setup.mdx

```diff
@@ -125,12 +125,12 @@ Here are the steps we’ll take:
 
         1. Sample commands if using previously tagged custom Docker image:
             ```bash
-            docker tag mageprod:latest [region]-docker.pkg.dev/[project_id]/[repository]/mageai:latest
+            docker tag mageai/mageai:latest [region]-docker.pkg.dev/[project_id]/[repository]/mageai:latest
             ```
 
             An example command could look like this:
 
-                docker tag mageprod:latest \
```

Comment:
This docker image name is referring to a past example in previous steps.

Reply:
Can we undo these changes please?

Reply:
sorry I don't know well about github pr. Is there anything I have to do?

Reply:
Yes, can you under these changes? They are suppose to be that way.

Reply:
past example in previous steps. -> could you share where is it? I just found there is only `mageai` name ..

Reply:
I updated the docs to call out the tag: https://github.com/mage-ai/mage-ai/pull/1797.

You can undo your changes at this line.